index.html

```diff
@@ -1755,6 +1864,100 @@ <h2>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        ProtocolHandler items
+      </h2>
+      <p>
+        Each <dfn>ProtocolHandlerItem</dfn> is an [=object=] that represents
```

Comment:
We no longer use IDL , having `ProtocolHandlerItem` looks a little odd compared to other similar objects in the spec... maybe: 
 
```suggestion
        Each <dfn>protocol handler description</dfn> is an [=object=] that represents
```

Or something...